docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1684,6 +1684,13 @@ type](#sec-user-defined-types). This is useful for
 [translation](#sec-psa-metadata-translation). The underlying built-in type must
 be a fixed-width unsigned bitstring (`bit<W>`).
 
+`Meter` & `DirectMeter` has an additional field `MeterCounterData` that holds
+the per color meter counter data. It has a granular list of the counters
+applicable to the possible color actions such as RED, YELLOW and GREEN.The unit
+of these counters are the same as the `counter_data` field. Note that certain
+implementations might not support these counters and it is okay to return an
+UNIMPLEMENTED error by the server in those circumstances.
```

Comment:
I feel like this should be justified more, probably by mentioning that the primary purpose of these counters is for debugging / troubleshooting.